checkpoint3/Program.cs

```diff
@@ -2,26 +2,101 @@
 using System.Linq;
 using System.Collections;
 using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore;
+
+
 
 namespace checkpoint3
 {
     class Program
     {
         static void Main(string[] args)
         {
+            DbContextOptionsBuilder<Utils> builder = new DbContextOptionsBuilder<Utils>();
+            builder.UseSqlServer("Server=(localdb)\\MSSQLLocalDB;Database=BlogDb;Trusted_Connection=true");
+            DbContextOptions<Utils> opts = builder.Options;
+            Utils context = new Utils(opts);
+
+            context.Database.EnsureCreated();
+
             ToDo todo1 = new ToDo();
-            Utils utils = new Utils();
+            ToDo todo2 = new ToDo();
+            ToDo todo3 = new ToDo();
+            ToDo todo4 = new ToDo();
 
-            utils.CreateToDo(todo1);
+            context.CreateToDo(todo1);
```

Comment:
Good job on Create, Edit and List All, but you are missing Delete, List by ID and not using EF to save your ToDos between application runs.